x-pack/plugins/security_solution/public/management/components/policy_endpoint_list_link.tsx

```diff
@@ -54,6 +55,9 @@ export const PolicyEndpointListLink = memo<
   }, [getAppUrl, filterByPolicyQuery, search]);
   const clickHandler = useNavigateByRouterEventHandler(toRoutePathWithBackOptions);
 
+  if (endpointCount === 0) {
```

Comment:
Yeah, I wonder why its a generic component (meaning: created this component under top-level `management/components`) instead of it just being a component under `management/pages/policy/**`.   @parkiino do you have other use cases that need this component outside of Policy related components?

I do like that this component controls the UX around whether a link should be shown or not, especially if it was going to be used generically in other places.  Maybe change the name of the competent to `PolicyEndpointCount`?